tests/sentry/integrations/slack/service/test_slack_service.py

```diff
@@ -170,6 +172,10 @@ def test_calls_handle_parent_notification_sdk_client(self, mock_handle):
         # check client type
         assert isinstance(mock_handle.call_args.kwargs["client"], SlackSdkClient)
 
+        assert len(mock_record.mock_calls) == 2
```

Comment:
will update this repetitiveness after my testutil will be merged from https://github.com/getsentry/sentry/pull/81825